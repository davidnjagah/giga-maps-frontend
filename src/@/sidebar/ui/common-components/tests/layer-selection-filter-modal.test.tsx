import { render } from '@testing-library/react';
import { testWrapper } from '~/tests/jest-wrapper';
import LayerSelectionFilterModal from '../layer-selection-filter-modal';

describe('Layer selection filter modal', () => {
  it('should render component', () => {
    const setOpen = jest.fn();
    const { asFragment } = render(testWrapper(<LayerSelectionFilterModal open={true} setOpen={setOpen} />));
    expect(asFragment).toMatchSnapshot();
  });
});
