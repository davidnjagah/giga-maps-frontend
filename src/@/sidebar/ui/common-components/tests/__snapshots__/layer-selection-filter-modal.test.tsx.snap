// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layer selection filter modal should render component 1`] = `[Function]`;
